tests/tests/datm_restart_cfsr

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
I'm surprised you didn't implement the restart test using a checkpoint restart which would have only required one additional test---using restarts from hour 12 of the 1d run, integrating for 12h and comparing to the existing 1d run.